git_machete/client.py

```diff
@@ -1049,7 +1047,7 @@ def print_line_prefix(branch_: LocalBranchShortName, suffix: str) -> None:
         sys.stdout.write(out.getvalue())
         out.close()
 
-        yellow_edge_branches = [k for k, v in edge_color.items() if v == AnsiEscapeCodes.YELLOW]
+        yellow_edge_branches = [k for k, v in sync_to_parent_status.items() if v == SyncToParentStatus.InSyncButForkPointOff]
         if yellow_edge_branches and warn_on_yellow_edges:
             if len(yellow_edge_branches) == 1:
                 first_part = f"yellow edge indicates that fork point for `{yellow_edge_branches[0]}` is probably incorrectly inferred,\n" \
```

Comment:
Albeit I'd still leave the `yellow` etc. naming in user-facing messages, for the sake of clarity